src/actions/dot/migrate/index.ts

```diff
@@ -75,12 +75,28 @@ const createCall = (api: ApiPromise, details: CallDetails): any => {
     case "claim": {
       return api.tx[section][method](args.index);
     }
-    case "addProxy": {
+    case "addProxy":
+    case "removeProxy": {
       return api.tx[section][method](args.proxy, args.proxy_type);
     }
     case "free": {
       return api.tx[section][method](args.index);
     }
+    case "mintClaim": {
+      return api.tx[section][method](args.who, args.value, args.vesting_schedule, args.statement);
+    }
+    case "forceVestedTransfer": {
+      return api.tx[section][method](args.source, args.target, args.schedule);
+    }
+    case "setBalance": {
+      return api.tx[section][method](args.who, args.new_free, args.new_reserved);
+    }
+    case "setValidatorCount": {
+      return api.tx[section][method](args.new);
+    }
+    case "scheduleNamed": {
```

Comment:
We will probably not include these (as they include the increases of the validator set size which will be handled differently in a new chain candidate). Some of them we may need to include, we will have to look at what each one is doing. We also need to keep in mind that the scheduled action might take place at a certain time of CC1's chain history that should be accurately reflected in the history of CC2.